README.md

```diff
@@ -23,20 +23,59 @@ For a better understanding of the plugin, here are some terms, which we will use
 If not already done install OSD as described in the [OSD installation guide](https://openseadragon.github.io/#download).
 After that, you can install the Plugin in multiple ways: 
 ### 1. Via npm/yarn/pnpm
-<!-- TODO: Add packagename when in npm! -->
 Use the installation-script for your prefered package manager from below:
 
-npm: `npm install XXX`
+npm: `npm install openseadragon-opacity-slider` or `npm i openseadragon-opacity-slider`
 
-yarn: `yarn add XXX`
+yarn: `yarn add openseadragon-opacity-slider` 
 
-pnpm: `pnpm add XXX`
+pnpm: `pnpm add openseadragon-opacity-slider` 
 
-### 2. Via download 
-Download the Plugin [here](https://github.com/WasMachenSachen/openseadragon-opacity-slider/archive/refs/heads/main.zip) and move it into your project directory. For size reasons only copy the `dist`-folder.
+After that import it into your JS- or HTML-File:
+```js
+import "<RELATIVE_PATH_TO_NODE_MODULES>/node_modules/openseadragon-opacity-slider/dist/openseadragon-opacity-slider.min.js"
```

Comment:
`import "openseadragon-opacity-slider";`
is fine

Reply:
or `import "openseadragon-opacity-slider/dist/openseadragon-opacity-slider.min.js";` for minifed version. but if you are importing we can assume that they bundling them self...